org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -356,21 +356,6 @@ public void issue217() {
 		testDiagnosticsFor(xml, d);
 	}
 
-	@Test
-	public void cvc_elt_3_1() throws Exception {

```

Comment:
why is this test removed?

Reply:
The test was wrong (and WTP XML validator doesn't validate that too)